package verify;

public class Ex4 {
	public static void main(String[] args) {
		System.out.println("4. throw에 대한 설명으로 틀린 것은 무엇입니까?\r\n"
				+ "1)예외를 최초로 발생시키는 코드이다.\r\n"
				+ "x2)예외를 호출한 곳으로 떠넘기기 위해 메소드 선언부에 작성된다.\r\n"
				+ "3)throw로 발생된 예외는 일반적으로 생성자나 메소드 선언부에 throws로 떠넘겨진다.\r\n"
				+ "4)throw 키워드 뒤에는 예외 객체 생성 코드가 온다.\r\n"
				+ "");
	}

}
